hbase-client/src/main/java/org/apache/hadoop/hbase/zookeeper/ReadOnlyZKClient.java

```diff
@@ -76,8 +83,14 @@ public final class ReadOnlyZKClient implements Closeable {
 
   private final int keepAliveTimeMs;
 
+  public static final HashedWheelTimer RETRY_TIMER = new HashedWheelTimer(
```

Comment:
Just reuse the one in AsyncConnectionImpl? Could pass it in when creating ReadOnlyZKClient.